Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1223,7 +1230,8 @@
     <RegEx find="([sS])avi?j?e([sš])" replaceWith="$1avje$2" />
     <RegEx find="\b([sSZz])avet" replaceWith="$1avjet" />
     <RegEx find="avremen" replaceWith="uvremen" />
-    <RegEx find="ažalj?eva" replaceWith="ažalijeva" />
+     <RegEx find="klj?eva" replaceWith="klijeva" />
+    <RegEx find="(aža|k)lj?eva" replaceWith="$1lijeva" />
     <RegEx find="\b([sS])ed([ei]|i[mš]|imo|it[ei]|eć[iaeu]|il[aeio]|iše)\b" replaceWith="$1jed$2" />
```

Comment:
Anything matched by `klj?eva` will also be matched by `(aža|k)lj?eva`, making the first of these two RegEx rules unnecessary.
```
PS> [regex]::Replace('kljeva', '(aža|k)lj?eva', '$1lijeva')
klijeva
PS> [regex]::Replace('kleva', '(aža|k)lj?eva', '$1lijeva')
klijeva
```